ragna/deploy/_ui/components/file_uploader.py

```diff
@@ -151,14 +144,16 @@ def perform_upload(self, event=None, after_upload_callback=None):
                     <img src="/imgs/cloud-upload.svg" width="24px" height="24px" />
                     <span><b>Click to upload</b> or drag and drop.<br /></span>
                     <div id='allowedDocuments'>
-                        ${allowed_documents_str}
+                        Allowed files: ${allowed_documents_str}
                     </div>
                     <input  type="file" 
                             name="fileUpload"
                             class="fileUpload" 
                             id="fileUpload" 
                             multiple="multiple" 
-                            onchange="${script('file_input_on_change')}" /> 
+                            onchange="${script('file_input_on_change')}" 
+                            accept="${allowed_documents_str}"
```

Comment:
This is not working for some reason. When looking at it with the browser console, I see

![image](https://github.com/Quansight/ragna/assets/6849766/bd88a64f-5c0f-448e-84ce-b7957cc09638)

`accept=""` is empty. Curiously, using the exact same syntax above to display the allowed files works just fine

![image](https://github.com/Quansight/ragna/assets/6849766/585d22e5-840b-4082-8ee6-4de1de27b01e)

Not sure what is going on here.

Reply:
Fix in d01ec2da2062cdb03cd18543c0bbcabe8a405248 by @pierrotsmnrd.